src/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
```

Comment:
I think it will be better to put all padding values into 1 option "padding"

Reply:
Oh yees, i forgot that i wrote them like this